quadruped_robot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 1000

  
  # gazebo_ros_control:
  #   pid_gains:
  #     LM : {p: 185000, d: 18000}


  # Driving Parts
  # effort_controllers / position_controllers
  # JointEffortController / JointPositionController
  LM_pos:
    type: effort_controllers/JointPositionController
    joint: LM
    pid: {p: 585000, d: 1000}
    # pid: {p: 285000, d: 8000}
    # pid: {p: 185000, d: 8000}
    # pid: {p: 45000, d: 800}
    # pid: {p: 30.0, i: 0.01, d: 15}
  RW_pos:
    type: effort_controllers/JointEffortController
    joint: RW
    pid: {p: 30.0, i: 0.01, d: 15}
  LW_pos:
    type: effort_controllers/JointEffortController
    joint: LW
    pid: {p: 30.0, i: 0.01, d: 15}
#velocity_controllers/JointVelocityController